### PR TITLE
Get it working

### LINKED ISSUE #2: Add Travis support
* enable the travis build
* add a `.travis.yml`

### LINKED ISSUE #5: Add code coverage
* after CI builds using Intern it should report code coverage
* upload code coverage to codecov.io

### LINKED ISSUE #10: Add git hooks
Restore the pre-commit git-hook that runs prettier

### PR DESCRIPTION
**Pull request by bryanforbes**
Modify codebase for new package structure

* Update imports to new relative locations
* Update intern configurations
* Update routing example
* Change to use `@dojo/scripts` for build
* Add npm scripts to invoke `@dojo/scripts` commands
* Enable TravisCI and Codecov.io

Resolves #2
Resolves #5
Resolves #10

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Thank You
+
+We very much welcome contributions to Dojo 2.
+
+Because we have so many repositories that are part of Dojo 2, we have located our [Contributing Guidelines](https://github.com/dojo/meta/blob/master/CONTRIBUTING.md) in our [Dojo 2 Meta Repository](https://github.com/dojo/meta#readme).
```
Comment (mwistrand) [inline .github/CONTRIBUTING.md]: This is just a stream of consciousness thought, but as we are moving to monorepo, does this copy need to change?

Comment (bryanforbes) [reply .github/CONTRIBUTING.md]: Yes. Can you file an issue to update this after the merge?

Comment (mwistrand) [reply .github/CONTRIBUTING.md]: Done: https://github.com/dojo/framework/issues/12

**Comment by bryanforbes** [comment]
I had to add a hack to the `i18n` code to get it to work with the types in `@types/globalize`, but we will be able to use actual paths once [the pull request](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/27271) I submitted to DefinitelyTyped is landed.

**Pull request merged**